lib/adapters/sequelize.js

```diff
@@ -85,7 +195,7 @@ module.exports = Bone => {
 
       const { where } = options;
       let spell = this._find(where)[`$${func}`](name);
-      if (options.paranoid === false) return spell.unscoped;
+      if (options.paranoid === false) return spell.unparanoid;
```

Comment:
I still am a bit hesitant about this `unparanoid` thing. Perhaps we can make all `_find()` calls unscoped and add `where({ deleted_at: null })` accordingly if paranoid is `false`?

The reason about my hesitation is, `unparanoid` is not even a word. `unscoped` is not that legit either but it still has appearances on the web. Perhaps `inparanoid`? 

Anyway, maybe we should have this discussed on Monday at  work.